### PR TITLE
[WIP] Add styling for top result

### PR DESCRIPTION
**Pull request by vanitabarrett**
Trello: https://trello.com/c/rzhhJI54/252-add-styling-to-high-relevance-content-on-business-readiness-finder-and-internal-search

To see what the styling will look like, add the 'document--top' class to a result list item.

Screen Shot 2019-03-12 at 10 16 42

**To-Do**
- Create a function that only shows this styling on the top, most relevant result per search (actual logic to be defined).
- Make sure this only applies to the business finder, not other finders
- Get the description to display for top items only.
- Make sure the top item is only shown when finder is sorted by Relevance.

### CODE REVIEW

File: app/presenters/result_set_presenter.rb [hunk 1]
```diff
@@ -49,13 +49,25 @@ def selected_filter_descriptions
     }.reject(&:empty?)
   end
 
+  def is_top_result?(score_a, score_b)
+    (score_a / score_b) > 1
```
Comment (DilwoarH) [inline app/presenters/result_set_presenter.rb]: We need to speak to Sam Dub about the scoring rule

File: app/views/finders/_document.mustache [hunk 2]
```diff
@@ -1,4 +1,5 @@
-<li class="document">
+<li class="document{{#document.top_result}} document--top{{/document.top_result}}">
+  <p>{{#document.top_result}}Most relevant result{{/document.top_result}}</p>
```
Comment (DilwoarH) [inline app/views/finders/_document.mustache]:
```suggestion
  {{#document.top_result}}<p>Most relevant result</p>{{/document.top_result}}
```

Comment (DilwoarH) [reply app/views/finders/_document.mustache]: The reason why I am suggesting this is because if there is no top results, it will add an empty `<p>` tag

**Comment by MahmudH** [comment]
Please refer to https://github.com/alphagov/finder-frontend/pull/974

**Pull request closed without merging**